documentation/modules/rn-2.2.adoc

```diff
@@ -98,3 +98,7 @@ If a Plan CR references storage, network, or VMs by name instead of by ID, the r
 .Log archive file includes logs of a deleted migration plan or VM
 
 If you delete a migration plan and then run a new migration plan with the same name or if you delete a migrated VM and then remigrate the source VM, the log archive file created by the {project-short} web console might include the logs of the deleted migration plan or VM. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2023764[*BZ#2023764*])
+
+.If a target VM is deleted, the VM migration status is still `Succeeded`
```

Comment:
Add "during migration" to the header: "If a target VM is deleted during migration, its migration status is "Succeeded" in the "Plan" CR"

Otherwise the title is not clear and the user might think that the target VM could be deleted at any time.

Change backticks in header to quotation marks ("). OpenShift docs does not allow [markup in headers](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings).